### DISCUSSION TITLE
Glosario: Traits

### DISCUSSION THREAD

**Discussion by Phosphorus-M** [RustLangES/rust-book-es#29] [Polls]
Hago esta encuesta por saber la opinion popular sobre si se debe traducir o no este keyword.

**Comment by xiron9090** [comment]
Pienso que debe mantenerse en inglés y sin traducir, pues cuando busques información tendrás mejores resultados usando la palabra Traits. También, a menudo, cuando hablas con otros desarrolladores o te comunicas en inglés, te será más fácil usar Traits.

**Reply by MiguelFernandezLorenzo** [reply]
Estoy de acuerdo @xiron9090, al final se trata de reducir las barreras innecesarias.

**Reply by Phosphorus-M** [reply]
Si finalmente se decidio dejarlo en traits XD
